release/release_tests.yaml

```diff
@@ -5348,6 +5348,59 @@
         cluster_env: app_config.yaml
         cluster_compute: data_ingest_benchmark_compute_gce.yaml
 
+- name: streaming_data_ingest_benchmark_100gb_gpu
+  group: data-tests
+  working_dir: nightly_tests/dataset
+
+  frequency: nightly
+  team: data
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: data_ingest_benchmark_compute_gpu.yaml
+
+  run:
+    timeout: 300
+    script: python data_ingest_benchmark.py --dataset-size-gb=100 --num-workers=4 --new_streaming --use-gpu
+    wait_for_nodes:
+      num_nodes: 3
```

Comment:
For my own understanding, when do we want to use `wait_for_nodes`? Don't think I've seen this before

Reply:
I believe it's because we don't want to include the time of autoscaling in the benchmark. We have this in other benchmarks as well.